experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -2924,13 +2924,14 @@ def test_get_versioned_schema_range_min_version_max_version_unsupported(self):
                 schemas=[unversioned_schema],
                 unsupported_in_range=False,
                 unsupported_versions=[],
+                supported_versions=[],
             ),
         )
 
     @parameterized.expand(
         [
-            None,
             packaging.version.Version("122.0.0"),
+            None,
```

Comment:
This was complaining to me locally when I tried to run it with `None` first 🤔

Reply:
What is the actual error?

Reply:
it was giving me something like "not able to create a test from None" but I can't repro it now o_o

Reply:
```
Traceback (most recent call last):
  File "/experimenter/./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.11/site-packages/django/core/management/commands/test.py", line 23, in run_from_argv
    super().run_from_argv(argv)
  File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/core/management/commands/test.py", line 55, in handle
    failures = test_runner.run_tests(test_labels)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/test/runner.py", line 722, in run_tests
    suite = self.build_suite(test_labels, extra_tests)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/test/runner.py", line 579, in build_suite
    tests = self.test_loader.loadTestsFromName(label)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/unittest/loader.py", line 214, in loadTestsFromName
    raise TypeError("don't know how to make test from: %s" % obj)
TypeError: don't know how to make test from: None
```

Reply:
I guess this is only failing when I'm running the individual test locally and it is parameterized, so no need to have it changed